Src/Plugins/ValidatorBase.cs

```diff
@@ -81,7 +93,7 @@ public static string CreateReturnValueForCompromisedAsset(string asset, string u
         /// set to null if no fingerprint can be computed that definitively identifies the secret.
         /// </param>
         /// <returns>Return the validation state.</returns>
-        protected abstract string IsValidHelper(
```

Comment:
>IsValidHelper [](start = 34, length = 13)

You can't change this, please revert. I will show you the pattern to use this base class.

Reply:
reverted this.

---
In reply to: [554482975](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554482975) [](ancestors = 554482975)